Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.BeIn.cs

```diff
@@ -49,4 +50,45 @@ public void When_asserting_subject_null_datetime_represents_a_specific_kind_it_s
                 .WithMessage("Expected subject to be in Utc, but found a <null> DateTime.");
         }
     }
+
+    public class NotBeIn
+    {
+        [Fact]
+        public void Date_is_not_in_kind()
+        {
+            // Arrange
+            DateTime subject = 5.January(2024).At(15.Hours().And(16.Minutes()).And(0.Seconds())).AsLocal();
+
+            // Act / Assert
+            subject.Should().NotBeIn(DateTimeKind.Utc);
+        }
+
+        [Fact]
+        public void Date_is_in_kind_but_should_not()
+        {
+            // Arrange
+            DateTime subject = 5.January(2024).At(15.Hours().And(16.Minutes()).And(0.Seconds())).AsLocal();
+
+            // Act
+            Action act = () => subject.Should().NotBeIn(DateTimeKind.Local);
+
+            // Assert
+            act.Should().Throw<XunitException>()
+                .WithMessage("Did not expect subject to be in Local, but it was.");
+        }
+
+        [Fact]
+        public void Date_is_null_on_kind_check()
+        {
+            // Arrange
+            DateTime? subject = null;
+
+            // Act
+            Action act = () => subject.Should().NotBeIn(DateTimeKind.Utc);
```

Comment:
```suggestion
            Action act = () =>
            {
                using var _ = new AssertionScope();
                subject.Should().NotBeIn(DateTimeKind.Utc);
            };
```

Reply:
Should I add this same thing also for `BeIn`?